### ISSUE TITLE
Project field in Bugzilla missing on new index (SOTRAR)

### ISSUE THREAD

**Issue by havardl** [mozilla/participation-metrics-org#205]
Hi @sanacl and @alpgarcia, 

As @edin-ogtal has previous mentioned, the new Bugzilla index currently has the project for all entries set to _Main_. For us, this has a high priority to get fixed, as we can't run any of our project centric analysis with this data source. 

cc @HerminaC @hmitsch for visibility

**Comment by alpgarcia** [comment]
There is a bug in the latest release causing this. It affects to all bugzilla data enriched from March 21st.

We marked it as Urgent Lane internally, so the team is working to fix it asap. We already found the version introducing the bug and our dev team is checking the changes we made to identify and fix the bug.

As @havardl confirmed us they are not using sotrar instance, we won't wait until 6pm to start enriching Bugzilla as we scheduled for updating identity information from SortingHat, but we will do it as soon as we have a release with the fix ready (related to https://github.com/mozilla/participation-metrics-org/issues/204#issuecomment-481990506).

Summarizing, you can expect to have the bug fixed at the same time as the identity information updated.

**Comment by alpgarcia** [comment]
There are currently 2 PRs been reviewed to fix the bug:
* https://github.com/chaoss/grimoirelab-sirmordred/pull/292
* https://github.com/chaoss/grimoirelab-elk/pull/610

Once they are merged we can create a new release and start the enrichment.

**Comment by alpgarcia** [comment]
ETA for having data ready in Bugzilla index is tomorrow morning as after deployment, index creation takes around 10 hours.

Areas of Code index is being regenerated in the meantime.

We will continue updating this ticket with status changes.

**Comment by havardl** [comment]
Thanks for the update @alpgarcia. Will this also resolve #203, or should we plan for something else?

**Comment by sduenas** [comment]
Hi @havardl , unfortunately we haven't fix this problem yet. It's a tricky one because as @alpgarcia commented, there are not fields in Bugzilla standard platform saying when a bug was closed. Mozilla guys have a custom field for it, though.

Moreover, as we assumed you were using the workarounds commented at https://github.com/mozilla/participation-metrics-org/issues/203#issuecomment-479496298 we didn't prioritize this bug correctly.

During next week it can be ready if we decided to use this custom field. Sorry for any trouble we might caused.
